### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by benjamincjackson** [COG-UK/grapevine#68]

**Comment by daveuu** [comment]
"Samples in different phylotypes share mutations with other sequences in the data set. An exception to this rule is that samples with different phylotypes may share a mutation if that mutation is a homoplasy (i.e. it arose multiple times on the tree)"

Is there a "not" missing in the first sentence above? Last section here: https://github.com/COG-UK/grapevine/blob/master/docs/lineages.md

**Comment by jtmccr1** [comment]
I don't think so, but the wording is confusing. Bringing in the line above it, would this be more clear?
 "Samples that have the same phylotypes are allowed differ from each other only by unique mutations. Samples in different phylotypes differ by at least one mutation found elsewhere in the data set."
We're trying to say the same thing two different ways, but that may be doing more harm than good. I'm happy just to have the first sentence if that is clear enough.

**Comment by daveuu** [comment]
Personally, the phrase "elsewhere in the dataset" might be confusing me - it seems a bit non-specific. Is this wording accurate (if not clear)?:
"Samples of the same phylotype are allowed to differ from each other only by unique mutations. Samples in different phylotypes differ by at least one mutation found only in their respective phylotypes."

**Comment by jtmccr1** [comment]
I like this much better.
